app/controllers/events_controller.rb

```diff
@@ -11,7 +20,7 @@ def create
     @event = current_user.created_events.build(event_params)
 
     if @event.save
-      respond_to { |format| update_events_list(format) }
+      respond_to { |format| update_events_list(format, @event) }
     else
       respond_to do |format|
         format.html { render :new, notice: 'Event was not created.' }
```

Comment:
remember there is no need to use `respond_to` and `format` here, since we are always responding to one format type

Reply:
agree, changed!